pkg/loki/modules.go

```diff
@@ -621,15 +621,6 @@ func (t *Loki) initQueryFrontendTripperware() (_ services.Service, err error) {
 }
 
 func (t *Loki) cacheGenClient() (generationnumber.CacheGenClient, error) {
```

Comment:
The cache gen client won't surface errors to the user if deletion is disabled. The behavior will be unchanged from what happens now when deletion is globally enabled but disabled for a user.